src/Microsoft.Health.Dicom.Core/Features/CustomTag/CustomTagEntry.cs

```diff
@@ -19,6 +21,14 @@ public CustomTagEntry(long key, string path, string vr, CustomTagLevel level, Cu
             Status = status;
         }
 
+        public CustomTagEntry(string path, string vr, CustomTagLevel level, CustomTagStatus status)
```

Comment:
As previous PR feedback, I created CustomTagStoreEntry standing for CustomTag entry in database, and CustomTagEntry is now only for customer input. The purpose is to isolate customer input and our internal data model. 
As per that change, Status and Key is removed.

But I think Status is also required when GetCustomTag Apis, so may be you could pull latest code and merge your changes here.